extensions/wikia/Recirculation/js/views/mixedFooter.js

```diff
@@ -46,14 +46,22 @@ define('ext.wikia.recirculation.views.mixedFooter', [
 			$wikiArticleHook = $('.mcf-card-wiki-placeholder'),
 			$sponsoredContentHook = $('.mcf-card-sponsored-content');
 
-        if (sponsoredContent) {
-            $sponsoredContentHook.replaceWith(
-                utils.renderTemplate(
-                    templates[templatePaths.sponsoredContent],
-                    $.extend(true, {}, sponsoredContent, { shortTitle: sponsoredContent.title.substring(0, 80) + '...' })
-                )
-            );
-        }
+		if (sponsoredContent) {
+			$sponsoredContentHook.replaceWith(
+				utils.renderTemplate(
+					templates[templatePaths.sponsoredContent],
+					$.extend(
+						true,
+						{},
+						sponsoredContent,
+						{
+							shortTitle: sponsoredContent.title.substring(0, 80) + '...',
+							attributionLabel: sponsoredContent.attributionLabel || 'Sponsored by'
```

Comment:
do we have translations for default "sponsored by" value? I think the assumption is we have it. If not lets notify stanley and ask if it is required or we'll be relying on the overrides put into the sponsored articles UI

Reply:
based on what Brett Bates wrote on Jira about having full control we won't translate that label, so I'd expect to have this field mandatory in form (or note that `Sponsored by` is the default somewhere)

Reply:
It is not mandatory in form and per my comment in the ticket it they know it will not be mandatory:
`so it would be one more input field in UI and if filled with text then use it instead of standard 'sponsored by', right?`

so we should make a note somewhere there

Reply:
I'll update a label in service